firefox-ios/firefox-ios-tests/Tests/ExperimentIntegrationTests.xctestplan

```diff
@@ -116,6 +116,7 @@
         "SyncUITests",
         "TabTrayTests",
         "TabTrayTests\/testAccessibility()",
+        "TabsTests",
```

Comment:
TopTabsTests wasn't excluded from ExperimentalIntegrationTEsts. It should not be run here.